Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h

```diff
@@ -11,6 +11,38 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#pragma mark - Error Parameters
+
+/// A wrapper objct that has a localized human-readable error message and the locale.
+@interface DBLocalizedUserMessage : NSObject
+
+/// Localized, human-readable text.
+@property(nonatomic, readonly, copy) NSString *text;
+//  IETF BCP 47 language tag of text locale.
+@property(nonatomic, readonly, copy) NSString *locale;
+
+- (instancetype)init NS_UNAVAILABLE;
+
+///
+/// DBLocalizedUserMessage full constructor.
+///
+/// @param text   The localized, human-readable text.
+/// @param locale The IETF BCP 47 language tag of text locale.
+///
+/// @return An initialized DBLocalizedUserMessage instance.
+///
+- (instancetype)initWithText:(NSString *)text
+                      locale:(NSString *)locale;
+
+///
+/// Description method.
+///
+/// @return A human-readable representation of the current DBLocalizedUserMessage object.
+///
+- (NSString *)description;
```

Comment:
It looks like this is the same as the `text` property. `description` is already a property of `NSObject` so it seems unnecessary to redeclare it here.

Reply:
yeah, it's unnecessary. I was trying to be consistent with the rest classes declared in this file. They all have `description` declared with the doc strings. Maybe people who worked this file wanted to have the explicit doc string for slightly better readability?

Reply:
Oh, I didn't realize it was like that elsewhere. SGTM.